build.sbt

```diff
@@ -80,6 +80,9 @@ libraryDependencies ++= Seq(
   "org.apache.spark" %% "spark-core" % testSparkVersion.value % "provided",
   "org.apache.spark" %% "spark-sql" % testSparkVersion.value % "provided",
   "org.apache.spark" %% "spark-hive" % testSparkVersion.value % "provided",
+  // added hadoop libs to test allowing to execute the tests locally (from within intellij)
+  "org.apache.hadoop" % "hadoop-common" % "3.3.1" % Test,
```

Comment:
these lib additions are causing the CI (Github Actions) build to fail

Reply:
oh yeah, just found out ;) do you know why? because for local testing I had to add those

Reply:
I'm afraid I don't know the niceties of this CI - I presume the different parallel runs are deliberately using different versions of Spark and Hadoop - so any of the jobs that need an older version of Hadoop libs are affected by this change

Reply:
libs are only used if not running in CI. Allows to test locally without modifying the build.sbt